data/sql/updates/pending_db_world/rev_1645574957481593300.sql

```diff
@@ -0,0 +1,1705 @@
+UPDATE `quest_template_locale` SET `Details`="$N，我们最近发现了天灾军团的一个新据点：破旧渡口。这处渡口就在瑟伯切尔的东方，有许多腐皮豺狼人和其他亡灵层层守卫着那里。$B$B天灾军团占据了那里，这其中一定有些什么阴谋。到破旧渡口去调查一下，看看为什么天灾军团会在那里。" WHERE `ID`=438 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="现在是把天灾军团彻底赶出银松森林的时候了。$B$B你必须再去一次那个岛，杀死图勒·鸦爪，把他的头颅提来见我！$B$B这个任务非常危险，但是你也应该知道它的重要性，$N。所以你只能成功，不能失败。" WHERE `ID`=442 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Objectives`="将图勒的头颅带回给瑟伯切尔的高级执行官哈德瑞克。" WHERE `ID`=442 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="致命的配方" WHERE `ID`=447 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="阿尔萨斯的手下越来越多。不过，只要再来一次新的瘟疫，我们就可以一劳永逸地同时根除天灾军团和人类。$b$b我的研究已经证明，将灰熊心脏中提取出的毒素和蜘蛛血混合在一起，就可以制成一种致命的毒药。从西北方的粘丝洞中的蜘蛛身上和银松森林中游荡着的熊身上采集一些样本回来，然后把它们送给幽暗城皇家药剂师学会里的大药剂师法拉尼尔。" WHERE `ID`=447 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="致命的配方" WHERE `ID`=450 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="其实你还能为皇家药剂师学会再做一些事情，$N。$b$b当初阿鲁高在整个森林中施展了他那可恶的魔法时，我们当中一位最有前途的成员——药剂师博拉尔德，也被他诅咒了。现在博拉尔德已经变成了一个忠于阿鲁高的傀儡，换句话说，他对我们已经彻底没用了。但是他的研究成果可能对我们非常有用，因为他在多年以前曾经研究过一种杀死了洛丹米尔湖中所有生物的瘟疫。$b$b将伯纳德的日记从他在焚木村旅店中的研究室里偷出来，然后把日记交给药剂师伦弗利尔。" WHERE `ID`=450 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="致命的配方" WHERE `ID`=451 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="根据博拉尔德的研究日记，采自湖岸爬行者和潜藏者身上的苔藓会对我们制造出这种瘟疫大有助益。这些野兽就栖息于东边的洛丹米尔湖中的群岛上。$b$b他还提到曾在一只邪鳍鱼人身上发现了一种非常稀有的硬瘤，显然这种瘤很难得到，博拉尔德也只找到了一个。$b$b收集一些苔藓和那种稀有的瘤，把它们带给幽暗城的大药剂师法拉尼尔。" WHERE `ID`=451 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="你看上去跟那帮腐皮豺狼人不一样，这真好。我都快绝望了……$B$B我叫奥拉瑞克，过去曾是图勒的大管家兼保镖。他用他的魔法“保护”了我，使我免受“亡灵瘟疫”的影响，但他的魔法也让我变成了这个半死不活的样子。而且，自从图勒创造了那群该死的腐皮豺狼人之后，他竟然砍了我的头……还把我的头给那群豺狼人当球踢！$B$B拜托，把我带到我的身体那儿去。那群豺狼人大概是把它埋在了芬里斯城堡的马厩旁……" WHERE `ID`=460 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Objectives`="将奥拉瑞克的头颅带到他的坟墓旁。" WHERE `ID`=460 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="<UNUSED>" WHERE `ID`=462 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="Other than you, we sent two scouts through Algaz.  Only one returned.  Here's what he discovered:$B$BThe Dragonmaws sent one of their Lieutenants, Maruk Wyrmscale, to organize a crew of orcs in Algaz and cut off the land route to Menethil Harbor!$B$BWe have to stop this, now.  Take out Maruk, and then report to Valstag Ironjaw in Menethil Harbor.$B$BMaruk is headquartered in a small cave, across a pond between the second and third tunnels of Algaz.  Menethil is through Algaz, then west along the road." WHERE `ID`=462 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Objectives`="Kill Maruk Wyrmscale$B$BReport to Valstag Ironjaw in Menethil Harbor." WHERE `ID`=462 AND `locale` = "zhCN";
```

Comment:
This is still English

Reply:
The English descriptions are correct. They are unused quests where that is the actual retail translation because not used - see https://cn.classic.wowhead.com/quest=462/unused.